library/oneview_server_profile.py

```diff
@@ -522,9 +522,14 @@ def __get_available_server_hardware_uri(self):
             return
 
         self.module.log(msg="Finding an available server hardware")
-        available_server_hardware = self.resource_client.get_available_servers(
-            enclosureGroupUri=enclosure_group,
-            serverHardwareTypeUri=server_hardware_type)
+        if self.oneview_client.api_version <= 1200:
+            available_server_hardware = self.resource_client.get_available_servers(
+                enclosureGroupUri=enclosure_group,
+                serverHardwareTypeUri=server_hardware_type)
+        else:
+            available_server_hardware = self.resource_client.get_available_targets(
+                enclosureGroupUri=enclosure_group,
+                serverHardwareTypeUri=server_hardware_type)['targets']
 
```

Comment:
This change is already made in one of the previous PR. Please check to avoid conflicts.

Reply:
done